cmd/minikube/cmd/logs.go

```diff
@@ -52,7 +55,23 @@ var logsCmd = &cobra.Command{
 	Short: "Returns logs to debug a local Kubernetes cluster",
 	Long:  `Gets the logs of the running instance, used for debugging minikube, not user code.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		logs.OutputOffline(numberOfLines)
+		var logOutput *os.File = os.Stdout
+		var err error
+
+		if fileOutput != "" {
+			logOutput, err = os.Create(fileOutput)
```

Comment:
So are we expecting an absolute or relative path here? Enforcing one or the other will make things less confusing for the user.

Reply:
It can be either. Isn't it standard to allow both relative and absolute paths for file parameters?

Reply:
we should allow user both, I image I would wanna do
```
minikube logs -f ~/Desktop/m.log
```
or

```
minikube logs -f /tmp/m.log
```